docs/app/pages/getting-started/getting-started.component.html

```diff
@@ -1,84 +1,160 @@
+                    </div>
+                </li>
+                <li>Install <a href="https://bower.io/">Bower</a>: <kbd>npm install -g bower</kbd></li>
+                <li>Install the UX Aspects bower package: <kbd>bower install ux-aspects#{{ version }}</kbd></li>
+            </ol>
+        </tab>
+        <tab heading="Direct Download">
+            <br>
+            <p>The UX Aspects resources (scripts, CSS, fonts, and images) can be downloaded from the
+                <a href="https://github.com/UXAspects/UXAspects/releases">GitHub releases page</a>.</p>
+        </tab>
+        <tab heading="Building from Source">
+            <br>
+            <ol>
+                <li>Clone the <a href="https://github.com/UXAspects/UXAspects">GitHub repository</a>.</li>
+                <li>Install the dependencies: <kbd>npm install</kbd></li>
+                <li>Build the project: <kbd>grunt build</kbd>. The resources (scripts, CSS, fonts, and images) can be found in the
+                    <code>dist</code> folder.
+                </li>
+            </ol>
+        </tab>
+    </tabset>
+
+    <br>
+
+    <h3>Applying the Stylesheet</h3>
+
+    <div class="row-divider"></div>
+
+    <p>The UX Aspects stylesheet can be applied in four different ways as outlined in the following tabs:</p>
+
+    <br>
+
+    <tabset>
+        <tab heading="CLI">
+            <br>
+            <ul>
+                <li>If using the CLI, the UX Aspects stylesheet can be added in the <code>angular-cli.json</code> file:
+                    <br><br>
+                    <uxd-snippet [code]="cliJson" language="javascript"></uxd-snippet>
+                </li>
+            </ul>
+        </tab>
+        <tab heading="Regular Webpack">
+            <br>
+            <ul>
+                <li>When using regular webpack, UX Aspects can be included in a stylesheet by using:
+                    <br><br>
+                    <uxd-snippet [code]="stylesheetCss" language="css"></uxd-snippet>
+                </li>
+            </ul>
+        </tab>
+        <tab heading="JavaScript/Typescript">
+            <br>
+            <ul>
+                <li>The UX Aspects stylesheet can also be included directly from JavaScript/TypeScript by using webpack:
+                    <br><br>
+                    <uxd-snippet [code]="webpackJs" language="javascript"></uxd-snippet>
+                </li>
+            </ul>
+        </tab>
+        <tab heading="Non-Angular Sites">
+            <br>
+            <ul>
+                <li>The UX Aspects stylesheet can be applied by adding it to the header of the site as follows:
+                    <br><br>
+                    <uxd-snippet [code]="headerHtml" language="html"></uxd-snippet>
+                    <p>Please note this is the only required step for non-Angular sites. However, this method is not recommended with
+                        the use of <code>node_modules</code>.</p>
+                </li>
```

Comment:
"... with the use of node_modules" - what does this mean?

I think this needs to read quite differently.

> If your application does not use Node or Webpack, you can copy the UX Aspects stylesheet into your site's assets directory from the following location, and include it in the head of your HTML.